server/src/metrics/edge_metrics.rs

```diff
@@ -0,0 +1,329 @@
+use ahash::{HashMap};
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+use tracing::info;
+use ulid::Ulid;
+use utoipa::ToSchema;
+
+use crate::types::BuildInfo;
+
+#[derive(Debug, Default, Clone, Copy, Deserialize, Serialize, ToSchema)]
+#[serde(rename_all = "camelCase")]
+pub struct LatencyMetrics {
+    pub avg: f64,
+    pub count: f64,
+    pub p99: f64,
+}
+
+impl LatencyMetrics {
```

Comment:
Default is already implemented for this struct, this shouldn't be needed

Reply:
Agreed.